clients/python/agentic-sandbox-client/agentic_sandbox/sandbox_client.py

```diff
@@ -109,16 +127,27 @@ def is_ready(self) -> bool:
         """Returns True if the sandbox is ready and the Gateway IP has been found."""
         return self.base_url is not None
 
-    def _create_claim(self):
+    @trace_span(f"{_trace_service_name}.create_claim")
```

Comment:
As noted, this f-string is evaluated once at import. To fix the naming issue, consider passing just the method name suffix (e.g., `"create_claim"`) to the decorator and having the decorator prepend the service name (retrieved from `self`) at runtime.